opsgenie_integration/src/main/java/com/atlassian/opsgenie/integration/CreateHandler.java

```diff
@@ -91,6 +93,9 @@
                     .build();
         }
 
+        logger.log("[CREATE] " + model.toString());
+        logger.log("[CREATE] PrimaryId: " + model.getPrimaryIdentifier());
```

Comment:
since its can contain API key we shouldn't log integration primary identifiers in crud handlers

Reply:
Removed, just printing out resource IDs